site/app/libraries/database/DatabaseQueries.php

```diff
@@ -1872,10 +1872,18 @@ public function getAllGradeablesIds() {
     }
 
     /**
-     * gets ids of all electronic gradeables
+     * gets ids of all electronic gradeables excluding assignments that will be bulk
+     * uploaded by TA or instructor.
+     *
+     * @return array
      */
     public function getAllElectronicGradeablesIds() {
-        $this->course_db->query("SELECT g_id, g_title FROM gradeable WHERE g_gradeable_type=0 ORDER BY g_grade_released_date DESC");
+        $this->course_db->query(
+          "SELECT g_id, g_title " .
+          "FROM gradeable INNER JOIN electronic_gradeable USING (g_id) " .
+          "WHERE eg_scanned_exam=FALSE and eg_has_due_date=TRUE " .
+          "ORDER BY eg_submission_due_date ASC"
+        );
```

Comment:
You can just open the `"` and leave it open to have a multiline string instead of doing it a string per line. We don't really care about the added whitespace in the query being sent to the DB, and it makes going back and editing the query a touch easier.

Reply:
Thank you for your feedback! Yes, I agree that only having a multiline string is easier for future modification. I fixed it in the most recent commits.